hack/jsonformat.go

```diff
@@ -13,8 +13,8 @@ func main() {
 		log.Fatal("Usage: jsonformat.go <filename>")
 	}
 
-	byt, err := ioutil.ReadFile(os.Args[1])
-	if err != nil {
+	byt, readErr := ioutil.ReadFile(os.Args[1])
+	if readErr != nil {
 		log.Fatalf("ERROR: Unable to read file: %v\n", os.Args[1])
```

Comment:
I kind of wanted to keep these pulls clean and shadow-related only. I'll add it in here but I'm not sure that sort of "small edit" is going to scale to larger packages with 30 shadowing instances.